candid/evm_rpc.did

```diff
@@ -166,7 +166,7 @@ type RejectionCode = variant {
   SysFatal;
   CanisterReject;
 };
-type FeeHistoryResult = variant { Ok : opt FeeHistory; Err : RpcError };
+type FeeHistoryResult = variant { Ok : FeeHistory; Err : RpcError };
```

Comment:
nit: here for a lack of a better place, in the PR title, isn't it a `fix` rather than a `feat`?

Reply:
Good point; I suppose it's a `fix!` due to it being a breaking change.